site/content/notes/updates-posts.md

```diff
@@ -0,0 +1,26 @@
+### 2022-09-14
+Hi, all!
+
+Thank you so much for your interest in Flowershow! Here are some of the new features we have implemented since the original post. Many more to come, so stay tuned! 😎🚀
+- First version of the CLI tool, that helps you install the Flowershow template, as well as build, and preview your website
```

Comment:
I'd suggest making these headings e.g. level 3 `###` e.g. (plus bonus for links to docs for that item)

```md
### First version of CLI tool

First version of the [docs/cli|CLI tool]], that helps you install the Flowershow template, as well as build, and preview your website

![[Pasted image 20220914185532.png]]

### Dark Light theme

[[/docs/theme|Dark/Light]] theme now supported -- plus ability to "pin" the theme.

...
```

Reply:
Good point! Will do!